### DISCUSSION TITLE
Newbee questions

### DISCUSSION THREAD

**Discussion by GerhardKreuzer** [marcokreeft87/room-card#325] [General]
Hi,
I tried this:

`type: custom:room-card
title: Keller
info_entities:
    - sensor.keller_lagerraum_flood_temperature
    - binary_sensor.keller_lagerraum_flood_flood
  `
And I get the name, the temperature and 'off' cause there is no flood in the cellar.
But I want to have a icon instead of the text 'off'

How can I do this?

Can I change the font size? (Please a newbe-answer)

Thanks a lot.

With best regards

Gerhard

**Comment by marcokreeft87** [comment]
Check the other discussion you started
